workflow/tests/hpxml_translator_test.rb

```diff
@@ -584,7 +589,7 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml, unit_multiplier)
       end
 
       # FIXME: Revert this eventually
-      next if message.include? 'Cannot currently handle an HPXML with multiple Building elements'
+      next if message.include? 'Cannot currently calculate utility bills based on detailed electric rates for an HPXML with unit multipliers or multiple Building elements'
```

Comment:
Wrap this in an `if` statement that only excludes the warning for the situations where it will occur.